hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.tsx

```diff
@@ -0,0 +1,288 @@
+                                            </TableHead>
+                                            <TableBody>
+                                                {pendingOrder.hardwareInTableRow.map(
+                                                    (row) => (
+                                                        <TableRow
+                                                            key={row.id}
+                                                            data-testid={`table-${pendingOrder.id}-${row.id}`}
+                                                        >
+                                                            <TableCell>
+                                                                <img
+                                                                    className={
+                                                                        styles.itemImg
+                                                                    }
+                                                                    src={
+                                                                        hardware[
+                                                                            row.id - 1
+                                                                        ]?.picture ??
+                                                                        hardwareImagePlaceholder
+                                                                    }
+                                                                    alt={
+                                                                        hardware[
+                                                                            row.id - 1
+                                                                        ]?.name
+                                                                    }
+                                                                />
+                                                            </TableCell>
+                                                            <TableCell>
+                                                                {
+                                                                    hardware[row.id - 1]
+                                                                        ?.name
+                                                                }
+                                                            </TableCell>
+                                                            <TableCell>
+                                                                {
+                                                                    hardware[row.id - 1]
+                                                                        ?.model_number
+                                                                }
+                                                            </TableCell>
+                                                            <TableCell>
```

Comment:
why does prettier do this ughhhh so much unnecessary whitespace

Reply:
haha true